### ISSUE TITLE
Window list not loading correct icons for some applications

### ISSUE THREAD

**Issue by PauloGDPeixoto** [linuxmint/Cinnamon#8155]
```
 * Cinnamon 4.0.5
 * Fedora 28
 * Intel Corporation 4th Gen Core Processor Integrated Graphics Controller
 * 64 bit
 ```

**Issue**
Some applications in the window list don't load the respective icon (screenshots attached), removing the window list from the panel and adding it again wield a warning (see other screenshot). Resizing the panel/switching icon theme brings back the correct icons, but as soon as the application is closed and started again, the issue happens again.
![screenshot from 2018-12-19 10-51-10](https://user-images.githubusercontent.com/20384915/50216068-b50f0800-037c-11e9-9d30-a8519346aea9.png)
![screenshot from 2018-12-19 10-46-23](https://user-images.githubusercontent.com/20384915/50216072-b7716200-037c-11e9-8e47-7aad1ef8cc59.png)

**Comment by brownsr** [comment]
There's not enough information here to investigate.  Could you please attach the .xsession-errors log, and details of what theme, icon set are in use.

**Comment by leigh123linux** [comment]
Try another theme, arc is dead in fedora since I abandoned it

https://src.fedoraproject.org/rpms/arc-theme/commits/master
https://github.com/horst3180/arc-theme/commits/master

**Comment by PauloGDPeixoto** [comment]
Switched theme, issue still occurs.

**Comment by leigh123linux** [comment]
Try this updated package

https://koji.fedoraproject.org/koji/buildinfo?buildID=1174380

**Comment by PauloGDPeixoto** [comment]
I switched themes and the issue persisted, so an updated theme package will hardly solve the issue.

In addition to that, I have a second laptop with exactly the same setup, software versions, etc... Where this issue does not occur.

**Comment by leigh123linux** [comment]
@PauloGDPeixoto  It works fine here in F29, I doubt I will ever use F28 again even though it's installed here as I've moved onto F30.

Try deleting ~/.cinnamon/configs/window-list@cinnamon.org/

rm -rf ~/.cinnamon/configs/window-list@cinnamon.org/

Also f28 has an update in testing.

https://bodhi.fedoraproject.org/updates/FEDORA-2018-af0019dd50


The last f28 package was pushed without any testing, it isn't my job to test.

https://bodhi.fedoraproject.org/updates/FEDORA-2018-232469ffea

**Comment by leigh123linux** [comment]
@brownsr I'm cool to close this as 'works for me', any ojections?

**Comment by jaszhix** [comment]
This was fixed in #8123, which was included in 4.0.6.
